src/CompletedTask.js

```diff
@@ -18,7 +18,7 @@ export default function CompletedTask({ completedItems, setCompletedItems }) {
         </div>
         <div className="completedTask-count">{completedItems.length}</div>
       </div>
-
+    { console.log(completedItems)}
```

Comment:
console should be removed after testing.

Reply:
console is removed in the new commit